patches/comm.sh.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/comm.sh b/comm.sh
+index ad2210c..61ae3e2 100644
+--- a/comm.sh
++++ b/comm.sh
+@@ -82,6 +82,9 @@ $AWK < ${srcdir}/comm.c >> comm.h '
+ '
+
+ $CC -E -I. -I${srcdir} ${srcdir}/comm.c > comm.cpp
++if [ "$(uname -s)" = "OS/390" ]; then
```

Comment:
Is this a bug in the compiler because it isn't tagging the output file as 1047? If so, should we open an issue? Is this using xlclang or clang?

Reply:
It's coming from /bin/sh. It's redirecting the stdout output from clang

Reply:
ok. should we enhance the check to say 'if not tagged, tag as 1047' in case we start using zsh or bash and it tags it properly?

Reply:
added